railties/test/application/middleware/sendfile_test.rb

```diff
@@ -58,7 +58,6 @@ def index
     test "files handled by ActionDispatch::Static are handled by Rack::Sendfile" do
       make_basic_app do |app|
         app.config.action_dispatch.x_sendfile_header = "X-Sendfile"
-        app.config.public_file_server.enabled = true
```

Comment:
I considered removing this in https://github.com/rails/rails/commit/1a5e80129c87a8610b0cdf9ee190615077611a7c, but, in the end, decided against it.  It doesn't hurt to have it in this test — it can serve as a declaration of intent.  And if for some reason we decide to flip the default again in the future, we won't have to come back and fix this test.